lib/kvdb/ikvdb.c

```diff
@@ -606,6 +676,63 @@ ikvdb_mclass_is_configured(struct ikvdb *const kvdb, const enum hse_mclass mclas
     return mpool_mclass_is_configured(self->ikdb_mp, mclass);
 }
 
+merr_t
+ikvdb_mclass_reconfigure(const char *kvdb_home, enum hse_mclass mclass, const char *path)
+{
+    struct kvdb_meta     meta, meta_orig;
+    struct kvdb_rparams  rp;
+    struct ikvdb        *kvdb;
+    merr_t err;
+
+    INVARIANT(kvdb_home && path);
```

Comment:
I think it is better to separate this out into two `INVARIANT()` calls. Up to you though

Reply:
I prefer this way...